inc/features/blocks/personalization/data/analytics.js

```diff
@@ -24,6 +24,13 @@ const reducer = function reducer( state, action ) {
 			};
 		}
 
+		case 'REMOVE_VIEWS': {
+			const key = `${ action.clientId }${ action.postId ? `-${ action.postId }` : '' }`;
+			const newState = { ...state };
+			delete newState.views[ key ];
+			return newState;
```

Comment:
Could also do

```suggestion
			const { [ key ]: deletedItem, ...newState } = state;
			return newState;
```

Reply:
Took me a while to wrap my head around that and still not sure if I do!